tedana/model/combine.py

```diff
@@ -8,26 +8,27 @@
 LGR = logging.getLogger(__name__)
 
 
-def make_optcom(data, t2s, tes, mask, combmode, verbose=True):
+def make_optcom(data, tes, mask, t2s=None, combmode='t2s', verbose=True):
```

Comment:
@emdupre I think we could probably consolidate these into one argument, but in the interest of clarity two is fine. If we move `t2s` so it's a keyword argument, then it can default to `None` (which works when `combmode` is `ste` but not when it's `t2s`). Then below we check the inputs to make sure they make sense. WDYT?

Reply:
Just to clarify: what would happen when `combmode` is `ste` if we make t2s the one (keyword) argument ? Or am I misunderstanding the suggestion ?

Reply:
That makes sense to me, unless we foresee adding other `combmode` options..  @handwerkerd might have thoughts on this !